test/azure/version-tolerant/Expected/AcceptanceTests/AzureParameterGroupingVersionTolerant/azureparametergroupingversiontolerant/aio/operations/_combine_operations.py

```diff
@@ -119,7 +119,7 @@ async def post_optional(
         error_map = {401: ClientAuthenticationError, 404: ResourceNotFoundError, 409: ResourceExistsError}
         error_map.update(kwargs.pop("error_map", {}))
 
-        request = build_post_optional_request(
+        request = build_parameter_grouping_post_optional_request(
```

Comment:
thanks for updating the version tolerant folder @msyyc! Now looking at the diffs, I just have 2 nits:
1. I think we don't have to call these operations files `_combine_operations.py`, I think `_operations.py` is better (this follows the pattern of the `models` files
2. I think we only need to add `additional_mark` if there are multiple operation group files. I don't feel super strongly about this nit, but I think it will help us generate a little less code, and since these request builders are embedded and hidden from users, even if someone adds an operation group later on and we add `additional_mark`, it won't be breaking. What are your thoughts?

Reply:
(1) For 1. in view of that there is existing file named `_operations.py`, so I name it with `_combine_operations.py`. But I think `_operations.py` is better and let me change it.
(2) For 2. according to my experience, almost all of service have multiple operation group files, so it is not very useful to add the additional judgement logic since it may never take effect. I advise to keep it as now to make the code clean asap.

Reply:
i think that's one of the big differences between data plane and mgmt plane, in data plane a lot of the services dont' actually have operation groups.

I don't care too much about this though because they're embedded, and changing them later won't be breaking, so I'll go ahead and merge this PR. Thanks so much for all of your hard work @msyyc!